bower.json

```diff
@@ -31,12 +31,14 @@
     "generated-docs"
   ],
   "dependencies": {
-    "purescript-halogen": "^5.0.0-rc.4",
-    "purescript-record": "^2.0.0"
+    "purescript-halogen": "^5.0.0-rc.9",
+    "purescript-halogen-hooks": "v0.4.0",
```

Comment:
If these aren't in the bower registry and `bower install` would fail, can we point these at GitHub links instead?

Reply:
Yeah. Done.